Installed/Driver/Task/BNF/dependencies.mk

```diff
@@ -0,0 +1,45 @@
+bnf.rdef: generic.uih product.uih \
+                /home/konstantinmeyer/github/pcgeos/Driver/Task/Common/task.ui
```

Comment:
This should probably be relative, not an absolute path that would only make sense on one particular person's machine.

Reply:
Yes, there is an issue in the generation tools when using mkmf/mkmf depend. For now manual correction after automatic generation of the files is the way to go.